flux/altinn-uptime/scripts/generate_targets.sh

```
@@ -0,0 +1,865 @@
+        echo "Failed to delete ServiceMonitor: $name"
+        return 1
+    fi
+}
+
+# Execute main workflow
+download_organizations
+load_configuration
+get_organizations
+
+# Initialize applied names file
+> "$TEMP_DIR/applied_names"
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)